src/pages/api/passengerType.ts

```diff
@@ -14,6 +18,36 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
         if (req.body.passengerType && passengerTypeValues.includes(req.body.passengerType)) {
             const { passengerType } = req.body;
 
+            if (passengerType === GROUP_REUSE_PASSENGER_TYPE) {
+                const savedGroups = getSessionAttribute(req, SAVED_PASSENGER_GROUPS);
+                if (!savedGroups) {
+                    throw new Error("Didn't have any saved groups but they should have been loaded on render");
+                }
+
+                const { reuseGroup } = req.body;
+                const reusedGroup = savedGroups.find(group => group.name === reuseGroup);
+                if (!reusedGroup || !reuseGroup) {
+                    const errors = [{ errorMessage: 'Please select a group to reuse', id: 'group-reuse' }];
```

Comment:
We shouldn't use the word 'please' in our error messages. We usually go for something like 'Choose a group to reuse' or 'Select a group to reuse'.

Reply:
Also. the id here needs to be passenger-type-group-reuse not group-reuse. This makes the rendered error message hyperlink to the appropriate area on the page.

Reply:
Ah, it should be `passenger-type-${GROUP_PASSENGER_TYPE}`

Reply:
👍